.gitignore

```diff
@@ -53,5 +53,5 @@ coverage.xml
 # Sphinx documentation
 docs/_build/
 
-# ansible-runner
-ansible_runner/artifacts/*
+# Ansible
+*.retry
```

Comment:
I find these can clutter the workspace, you can turn them off or set a global configuration spot for them e.g. ~/.cache/retries